documentation/source/requirements.txt

```
@@ -0,0 +1,2 @@
+sphinxcontrib.programoutput
+recommonmark
```

```suggestion
sphinx
sphinx_rtd_theme
sphinxcontrib.programoutput
recommonmark
```
As per setup.py, are `sphinx` and `sphinx_rtd_theme` also needed here? Or are the installation of those packages handled elsewhere?

I ask because when building the docs locally using `make html`, I had to manually install them  before it would build.

---

> As per setup.py, are sphinx and sphinx_rtd_theme also needed here? Or are the installation of those packages handled elsewhere?

needed by readthedocs, to avoid https://github.com/neuropoly/spinalcordtoolbox/pull/2814#issuecomment-669168870

---

full disclosure-- however i haven't tested if sphinx is needed specifically here-- it doens't hurt and i have no time to test-- so i leave it here for now (feel free to experiment)